execution_engine/src/core/engine_state/mod.rs

```diff
@@ -795,6 +810,26 @@ where
                 .write(locked_funds_period_key, value);
         }
 
+        if let Some(new_round_seigniorage_rate) = upgrade_config.new_round_seigniorage_rate() {
```

Comment:
The spec should be updated to include this (plus new_auction_delay and new_locked_funds_period immediately above it in the logic).

Reply:
Did this